### ISSUE TITLE
what's the gain of BCEBlurWithLogitsLoss?

### ISSUE THREAD

**Issue by zfli-xdu** [ultralytics/yolov5#1030]
https://github.com/ultralytics/yolov5/blob/5a9c5c1d3b02c8796e6af153d57af5821d8db0d6/utils/general.py#L454-L468

hi, I dont understand what is BCEBlurWithLogitsLoss about? When should i use it?

**Comment by Aktcob** [comment]
dx = pred - true  # reduce only missing label effects 
         # dx = (pred - true).abs()  # reduce missing label and false label effects

**Comment by glenn-jocher** [comment]
This is simply an experimental loss function. The intention is shown in the comments of the lines pasted by @Aktcob.

**Comment by zfli-xdu** [comment]
> ```
>      dx = pred - true  # reduce only missing label effects 
>      # dx = (pred - true).abs()  # reduce missing label and false label effects 
> ```

thanks, but i still dont understand this formula

**Comment by zfli-xdu** [comment]
> This is simply an experimental loss function. The intention is shown in the comments of the lines pasted by @Aktcob.
does this formula have some detailed explanations? Any related references?

**Comment by glenn-jocher** [comment]
@ZhengfengLee you can plot the loss when true = 1 and true = 0 to see the effect. The idea was to reduce the effects of false negatives (missing labels), which can occur often in COCO and other datasets, but I was not able to produce improved training with the current implementation unfortunately.

**Comment by zfli-xdu** [comment]
> @ZhengfengLee you can plot the loss when true = 1 and true = 0 to see the effect. The idea was to reduce the effects of false negatives (missing labels), which can occur often in COCO and other datasets, but I was not able to produce improved training with the current implementation unfortunately.

thanks for your help, i will run more tests to understand it
